lib/command/winRate.js

```diff
@@ -68,7 +68,7 @@ function twrFn(msg, authorRecord, tankName) {
 				});
 			}
 
-			return this.wotblitz.tanks.stats(record.account_id, null, tankIds, null, ['tank_id', 'all.battles', 'all.wins'])
+			return this.wotblitz.tanks.stats(record.account_id, null, tankIds, null, ['tank_id', 'all.battles', 'all.wins', 'all.damage_dealt'])
```

Comment:
I am pretty sure this makes you the first user of wotblitz. If you can remember, give me some idea how long it took you to identify what change to make here.